service/server.py

```diff
@@ -123,17 +121,17 @@ def cookies():
     return _cookies_page()
 
 
+@app.route('/terms-and-conditions', methods=['GET'])
+def terms_and_conditions():
+    return _terms_and_conditions_page()
```

Comment:
Could simply be:

return render_template('terms_and_conditions.html')

but I'm not that bothered.